Taylor_series_Repo/Taylor_series_Horner_rule.c

```diff
@@ -0,0 +1,61 @@
+/*Description of Algorithm:-
+
+Horner's rule for polynomial division is an algorithm used to simplify the process of evaluating a polynomial f(x) at a certain value x = x0 by dividing the polynomial into monomials (polynomials of the 1st degree). Each monomial involves a maximum of one multiplication and one addition processes. The result obtained from one monomial is added to the result obtained from the next monomial and so forth in an accumulative addition fashion. This division process is also known as synthetic division.
+
+To explain the above, let is re-write the polynomial in its expanded form;
+
+f(x0) = a0 + a1x0 + a2x02 + ... + anx0n
+This can, also, be written as:
+
+f(x0) = a0 + x0(a1 + x0(a2 + x0(a3 + ... + (an-1 + anx0)....)
+The algorithm proposed by this rule is based on evaluating the monomials formed above starting from the one in the inner-most parenthesis and move out to evaluate the monomials in the outer parenthesis.
+
+The algorithm is executed following the below steps:
+
+1. Set k = n
+2. Let bk = ak
+3. Let bk - 1 = ak - 1 + bkx0
+4. Let k = k - 1
+5. If k ≥ 0 then go to step 3
+Else End
+*/
+/*Example:-
+This algorithm can, also, be graphically visualized by considering the 5th degree polynomial given by:
+
+f(x) = a0 + a1x + a2x2 + a3x3 + a4x4 + a5x5
+which can be evaluated at x = x0 by re-arranging it as:
+
+f(x0) = a0 + x0(a1 + x0(a2 + x0(a3 + x0(a4 + a5x0))))*/
+
+//=====*****=====*****=====*****======******//
+
+
+/*Time-Complexity:O(n)*/
+
+
+//****************************************//
+
+
+#include<stdio.h>
+double e(int x,int n)
+{
+
+    static double s;
+    if(n==0)
+    {
+

```

Comment:
Remove line

Reply:
> @Anshuvats Please remove the extra file in the pr. (there are 8 files in the pr)

@somya-kapoor, I have already squashed this into one commit. Then, please help me, how can I remove this file.

Reply:
> Remove line
@somya-kapoor  If I remove this line, the base case will delete. then, what is the termination condition of my recursive solution code? If you have some idea about this. please, help me.

Reply:
I am talking about line 46 if you removed it nothing will happen to the code.

Reply:
Please see https://stackoverflow.com/questions/12481639/remove-files-from-git-commit/12481977
If you still face the issue, we can discuss it on slack.

Reply:
> Please see https://stackoverflow.com/questions/12481639/remove-files-from-git-commit/12481977
> If you still face the issue, we can discuss it on slack.

 I am not able to squash the commit. please, help me @somya-kapoor. please, reach out me on slack.